numpyro/distributions/gof.py

```diff
@@ -186,7 +186,7 @@ def density_goodness_of_fit(samples, probs, plot=False):
     if len(samples) <= 100:
         raise InvalidTest("imprecision; use more samples")
 
-    index = np.argsort(samples, 0, kind="quicksort")
+    index = np.argsort(samples, 0, kind="stable")
```

Comment:
This seems to be unnecessary. I think you can ping scipy version to 1.6.3. Newer versions seem to have bug (see https://github.com/scipy/scipy/issues/14953).

Reply:
To suppress a new deprecation warning in CI, could you help me change `np.int` in line 148 to `int`. Thanks!